DESCRIPTION.md

```diff
@@ -12,6 +12,9 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
   - Added support for `use_logical_type` in `write_pandas`.
   - Removed dependencies on pycryptodomex and oscrypto. All connections now go through OpenSSL via the cryptography library, which was already a dependency.
+  - Added the `backoff` argument to `snowflake.connector.connect` allowing for configurable backoff policy between retries of failed requests. See available implementations in the `time_util` module.
```

Comment:
I only see `TimeoutBackoffCtx` in the `time_util`, I assume this is the base class, are we missing other implementations in the module?

also I think we'd better call it out explicitly what's the implementations here because `time_util` has some other classes, module variables, methods, etc.

Ideally we can say, available implementations of the backoff policy are:
- snowflake.connector.time_util.XXX
- snowflake.connector.time_util.YYY

this gives us 2 benefits:
-  being clear on what can be used by users
- only the classes called out the public apis, we leave us some room for future change for other apis in the module